src/components/task/create-new/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from "react";
+import { useForm } from "react-hook-form";
```

Comment:
I strongly recommend, while you're in the learning phase of React, it's important to understand the fundamentals and core concepts of the library and its form handling capabilities by manually implementing form handling without relying on a library. It'll help you gain a deeper understanding of how React handles form state, form submission, validation etc. and build a strong foundation.